sympy/series/series.py

```diff
@@ -10,3 +13,133 @@ def series(expr, x=None, x0=0, n=6, dir="+"):
     """
     expr = sympify(expr)
     return expr.series(x, x0, n, dir)
+
+
+def lagrange_inversion_theorem(eq, no_of_terms = 3):
+    """
+    The Lagrange inversion theorem (or Lagrange inversion formula, which we abbreviate
+    as LIT), also known as the Lagrange--Bürmann formula, gives the Taylor series
+    expansion of the inverse function of an analytic function.
+
+    The theorem was proved by Lagrange (1736--1813) and generalized by the German
+    mathematician and teacher Hans Heinrich Bürmann ( --1817), both in the late 18th
+    century.
+
+    The Lagrange inversion formula is one of the fundamental formulas of combinatorics.
+    In its simplest form it gives a formula for the power series coefficients of the
+    solution f(x) of the function equation f(x) = xG(f(x)) in terms of coefficients of
+    powers of G.
+
+    Theorem: Suppose z is defined as a function of w by an equation of the form
+
+    f(w) = z
+
+    where f is analytic at a point a and f'(a) is not equal to 0.
+    Then it is possible to invert or solve the equation for w in the form of a series:
+
+    w = a + \sum_{n = 1}^{oo} (g_n*(z - f(a))**n)/n!
```

Comment:
Use `.. math::` here so that it formats correctly in the docs. I would suggest adding this function to the Sphinx docs so that you can see if it renders correctly.

Reply:
Yes will do so. That part is yet to be corrected. I wasn't sure about the way the formula is been shown in the documentation (for now). So, this is a point that has to be taken care of surely. I was trying to set all other parts correctly before I do that (It's one of the easier parts, so can be done in the end). Thanks for this. Also, I need to get around with Sphinx docs (I haven't got the good fortune of yet interacting with it). Please do not set this point as ' resolved ' right now.